data/operators.csv

```diff
@@ -266,6 +266,7 @@ INTERSPACE-MK,cloud,,unsafe,200899,5200
 Monkeybrains,ISP,,unsafe,32329,5215
 BroadbandGibraltarLtd.,ISP,,unsafe,34803,5285
 AltusHost,cloud,,unsafe,51430,5306
+Kingston Communications,ISP,,unsafe,12390,5388
```

Comment:
You are right about them signing their prefixes. (Sorry about the back-and-forth on the changes, it's on me!)
```suggestion
Kingston Communications,ISP,signed,unsafe,12390,5388
```